core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -691,6 +722,48 @@ public void minute() {
     assertEquals("minute(\"2020-08-17 01:02:03\")", expression.toString());
   }
 
+  public void testMinuteOfDay(String date, int value) {
```

Comment:
`private`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/189/commits/5778d63fcffc41f5c029f2cbb79bc85250441f7b